app/helpers/inthenews_helper.rb

```diff
@@ -53,4 +53,15 @@ def link_simple_li_item(assoc_rec, path)
     end
   end
 
+  def list_search_filters(label)
+    filters = []
+    params.each do |k, v|
+      next if [ "action", "controller" ].include?(k)
+      v = @person.name if k == "person"
+      filters << "#{v} (#{k.titleize.downcase})"
+    end
+    text = filters.join(", ")
+    "#{label.pluralize} filtered by #{text}"
```

Comment:
I think we ought to add a ternary here so it doesn't display anything if `text` is blank, e.g.
```ruby
text.blank? ? "" : "#{label.pluralize} filtered by #{text}"
```

I see on the Browse All Poets page "Poets filtered by" followed by nothing, so it looks a little odd :)

Reply:
Whoops yep!